### PR TITLE
[CHK-2007]: Add test to ensure pre-filled profile and shipping data pickup purchases are completed

### PR DESCRIPTION
**Pull request by jeffersontuc**
Recently we solved a known issue where the user couldn't finish an order when the needed information, such as profile and shipping data, would be filled using the API directly, without using our UI.

This PR adds a test case to ensure this problem doesn't happen anymore in the future.

In order to know more details about the problem, you can read this [JIRA card](https://vtex-dev.atlassian.net/browse/CHK-1728).

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import test from './models/Pickup - Pre-filled profile and shipping data'
+
+test('vtexgame1')
```
Comment (lucasecdb) [inline tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js:3]: should we test on other accounts too? such as `vtexgame1geo` and `vtexgame1nolean`

File: tests/shipping/models/Pickup - Pre-filled profile and shipping data.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      visitAndClearCookies(account)
+    })
+
+    it('with only pickup', () => {
+      const email = getRandomEmail()
+
+      const profileData = {
+        email,
+        firstName: 'Paulo',
+        lastName: 'Pipoco',
```
Comment (lucasecdb) [inline tests/shipping/models/Pickup - Pre-filled profile and shipping data.js:23]: 😂

**Review by lucasecdb — APPROVED** [review]
LGTM! just left a question on whether we should test on more accounts or not

File: tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js (hunk 1 above)
Comment (jeffersontuc) [reply tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js:3]: hmm I think it's worthy! :onit:

Comment (jeffersontuc) [reply tests/shipping/Pickup - Pre-filled profile and shipping data - vtexgame1.test.js:3]: Just added the test for `vtexgame1geo` and `vtexgame1nolean` as well.

**Pull request merged**